src/mumble/About.cpp

```diff
@@ -28,9 +28,9 @@ AboutDialog::AboutDialog(QWidget *p) : QDialog(p) {
 	for(int i = 0; !licenses3rdParties[i].isEmpty(); ++i)
 	{
 		qtb3rdPartyLicense->append(QString::fromLatin1("<h3>%1 (<a href=\"%2\">%2</a>)</h3><pre>%3</pre>")
-				.arg(QString::fromLatin1(licenses3rdParties[i].name))
-				.arg(QString::fromLatin1(licenses3rdParties[i].url))
-				.arg(QString::fromLatin1(licenses3rdParties[i].license)));
+				.arg(Qt::escape(QString::fromLatin1(licenses3rdParties[i].name)))
```

Comment:
`Qt::escape` [is obsolete](http://doc.qt.io/qt-5/qt-obsolete.html#escape).

Use [`QString::toHtmlEscaped()`](http://doc.qt.io/qt-5/qstring.html#toHtmlEscaped) instead.